jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/SysinfoCpuTime.java

```diff
@@ -29,19 +29,23 @@
 	private int numberOfCpus; /* should be at least 1 */
 	private int status;
 	private long processCpuTime; /* in ns. The process Cpu Time, used for getProcessCpuLoad() */
-
+	private int cpuLoad;
+	
 	/**
 	 * @param timestamp time in nanoseconds from a fixed but arbitrary point in time 
 	 * @param cpuTime time spent in user, nice, and system/privileged state by all CPUs.
 	 * @param numberOfCpus number of CPUs as reported by the operating system.
 	 * @param status status as reported by the JNI native
+	 * @param status cpuLoad ZOS system cpu utilization
 	 */
-	SysinfoCpuTime(long timestamp, long cpuTime, int numberOfCpus, int status) {
+	SysinfoCpuTime(long timestamp, long cpuTime, int numberOfCpus, int status, int cpuLoad) {
 		super();
 		this.timestamp = timestamp;
 		this.cpuTime = cpuTime;
 		this.numberOfCpus = numberOfCpus;
 		this.status = status;
+		this.cpuLoad = cpuLoad;
+
```

Comment:
Extra space here.

Reply:
Fix in 5edae7db32e8a09931cf979411bb18bf46ad95b2